types/underscore/index.d.ts

```diff
@@ -4579,10 +4562,15 @@ declare module _ {
         findLastIndex(predicate: _.ListIterator<T, boolean> | {}, context?: any): number;
 
         /**
-        * Wrapped type `any[]`.
-        * @see _.sortedIndex
-        **/
-        sortedIndex(value: T, iterator?: (x: T) => any, context?: any): number;
+         * Uses a binary search to determine the index at which the value should be inserted into the wrapped list in order
```

Comment:
I suddenly realize these lines are rather long. Maybe this doesn't belong in the current PR, but in general, I'd prefer to keep lines within 80 columns whenever possible (the combinatorial tests that we discussed before are an exception, of course).

Reply:
I agree that this summary comment in particular is a bit longer than others; I'd be happy to update that while I'm in here.

Reply:
Summary comments for the functions I touched have been updated to be less than 80 characters long 😄